python/graphstorm/run/launch.py

```diff
@@ -324,7 +325,7 @@ def construct_torch_dist_launcher_cmd(
     )
 
 def wrap_dist_remap_command(
-        udf_command: str,
+        udf_command: list,
```

Comment:
Typing hinting generics for standard collections (e.g. `list`, `dict`) were introduced in Python 3.9 ([PEP-585](https://peps.python.org/pep-0585/)), to support Python>=3.7 use:

```suggestion
        udf_command: List[str],
```